tests/Unit.Tests/Client/HttpClientWrapperTests.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Text;
+using GovUk.Education.SearchAndCompare.Domain.Client;
+using Moq;
+using Moq.Protected;
+using NUnit.Framework;
+
+namespace GovUk.Education.SearchAndCompare.Api.Tests.Unit.Tests.Client
+{
+    [TestFixture]
+    public class HttpClientWrapperTests
+    {
+        private HttpClientWrapper sut;
+        private  Mock<HttpMessageHandler> mockMessageHandler;
+        [SetUp]
+        public void SetUp()
+        {
+            mockMessageHandler = new Mock<HttpMessageHandler>();
+            sut = new HttpClientWrapper(new HttpClient(mockMessageHandler.Object));
```

Comment:
sut sucks as a name imho, call it httpClientWrapper

Reply:
disgree that is the subject under test